src/components/locations/components/LocationById.js

```diff
@@ -29,17 +32,15 @@ const LocationById = ({ match, history }) => {
         </Typography.Text>
     );
     const renderImages = locImages => locImages.map((element, i) => (
-        !locationLoading ? (
-            <img
-                key={element}
-                alt={i}
-                src={element}
-                style={{
-                    objectFit: 'cover',
-                }}
-            />
-        ) : ''
-
+        <img
+            key={element}
+            alt={i}
```

Comment:
alt shouldn't be a random index, it should be a description of the image... simply doing 
`"{location.name} photo"` is better than using an index